assets/addon_browser/addons.conf

```diff
@@ -758,4 +758,15 @@
       "github": "https://github.com/TechnicJelle/BlueMapWebAddons/tree/main/copy-coordinates"
     }
   }
+  {
+    name: "BluemapFrontiers"
+    description: "MapFrontiers integration for BlueMap"
+    author: "ErdbeerbaerLP"
+    apiVersion: ""
```

Comment:
Please fill this in

(You're using API 2: https://github.com/ErdbeerbaerLP/BluemapFrontiers/blob/e53c68a57062713cae73e3bfb3e1487604b69c89/common/build.gradle#L24 )

Reply:
Added API version 2 with https://github.com/BlueMap-Minecraft/BlueMapWiki/pull/85/commits/71eda2850260b766c3d979f99111ed54c31faa84